import/popups/BlockedPopupDialog.qml

```diff
@@ -0,0 +1,63 @@
+/****************************************************************************
+**
+** Copyright (c) 2021 Open Mobile Platform LLC.
+**
+****************************************************************************/
+
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import QtQuick 2.0
+import Sailfish.Silica 1.0
+
+Dialog {
+    id: dialog
+
+    property alias host: blockedPopup.host
+    property alias rememberValue: blockedPopup.rememberValue
+
+    BlockedPopupInterface {
+        id: blockedPopup
+
+        anchors.fill: parent
+        rememberValue: remember.checked
+
+        Column {
+            width: parent.width
+            spacing: Theme.paddingMedium
+
+            DialogHeader {
+                //: Allow the site to open a popup window
+                //% "Allow"
+                acceptText: qsTrId("sailfish_components_webview_popups-he-load_popup")
+
+                //: Deny the site from opening a popup window
+                //% "Deny"
+                cancelText: qsTrId("sailfish_components_webview_popups-he-block_popup")
+            }
+
+            Label {
+                x: Theme.horizontalPageMargin
+                width: parent.width - x * 2
+
+                //: %1 is the site that wants to load a popup
+                //% "The site %1 has requested to open a new tab. Would you like to allow it to open new tabs?"
+                text: qsTrId("sailfish_components_webview_popups-la-popup_request").arg(blockedPopup.host)
```

Comment:
There is a already have blocked the opening new tab, and it isn't request, may be should change this message?
What do you think?

Reply:
Good point, yes.  Perhaps something like:
`"The site %1 attempted to open a new tab, which has been blocked.  Would you like to allow it to open new tabs?"`
instead?

Reply:
It's ok for me

Reply:
Done, thanks.